### PR TITLE
[REEF-221]  Fix NuGet publish issue

### PR DESCRIPTION
**Pull request by jwang98052**
This PR includes:

The fixes for pushing NuGet to NuGet repository automatically for Bridge Nugets
Update the NuGet number to next one that is 9
Changed the jar file location in the NuGet to use "content"

JIRA: REEF-221. (https://issues.apache.org/jira/browse/REEF-221)

Author: Julia Wang  Email: jwang98052@yahoo.com

### CODE REVIEW

**Comment by markusweimer** [comment]
I'll look into this.

File: lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj [hunk 1]
```diff
@@ -93,5 +93,21 @@ under the License.
...
+            <PushCommand Condition="'$(NuGetApiKey)' == '' OR '$(NuGetApiKey)' == '*Undefined*'">$(NuGetCommand) push @(PackageFile) -Source $(NuGetRepository)</PushCommand>
+            <PushCommand Condition="'$(NuGetApiKey)' != '' AND '$(NuGetApiKey)' != '*Undefined*'">$(NuGetCommand) push @(PackageFile) $(NuGetApiKey) -Source $(NuGetRepository)</PushCommand>
+        </PropertyGroup>
+
+       <Exec
+            Condition="'$(BuildPackage)' == 'true'"
+            Command="$(PushCommand)"
+            LogStandardErrorAsError="true">
+        </Exec>
+	<Message Text="xxx PackageFile is [$(PackageFile)]"/>
```
Comment (markusweimer) [inline lang/cs/Org.Apache.REEF.Bridge.JAR/Org.Apache.REEF.Bridge.JAR.csproj]: This is a leftover from debugging, right? It should be removed.

File: lang/cs/build.props [hunk 2]
```diff
@@ -52,9 +52,10 @@ under the License.
   <!-- REEF NuGet properties -->
   <PropertyGroup>
     <IsSnapshot>true</IsSnapshot>
-    <SnapshotNumber>7</SnapshotNumber>
+    <SnapshotNumber>9</SnapshotNumber>
     <PushPackages>false</PushPackages>
-    <NuGetRepository>https://www.nuget.org</NuGetRepository>
+    <!--NuGetRepository>https://www.nuget.org</NuGetRepository-->
+    <NuGetRepository>http://localhost:46824</NuGetRepository>
```
Comment (markusweimer) [inline lang/cs/build.props]: Please don't commit with references to localhost.

File: lang/cs/build.props [hunk 3]
```diff
@@ -94,10 +97,11 @@ under the License.
       <Using Namespace="System.Xml.Linq"/>
       <Code Type="Fragment" Language="cs">
         <![CDATA[
-Version = XDocument.Load(Path.Combine(ProjectFolder, "pom.xml")).Descendants()
+          Version = XDocument.Load(Path.Combine(ProjectFolder, "pom.xml")).Descendants()
             .Where(x => x.Name.ToString().Contains("version"))
             .FirstOrDefault().Value;
-]]>
+          NugetVersion = Version.Replace("-incubating","") + "-" + SnapshotNumber;
```
Comment (markusweimer) [inline lang/cs/build.props]: Please do not remove the `-incubating`. All our releases will have to have it till we graduate.

**Comment by markusweimer** [comment]
@jwang98052 I did a pass.

File: lang/cs/build.props (hunk 3 above)
Comment (jwang98052) [inline lang/cs/build.props]: "incubating" has been removed in the current script in Git. We can have a separate JIRA to clean it up.

Comment (jwang98052) [inline lang/cs/build.props]: I tried it, got error "The special version part cannot exceed 20 characters" from NuGet.exe. Looks like our version number is too long id adding incubating.

File: lang/cs/build.props (hunk 2 above)
Comment (jwang98052) [inline lang/cs/build.props]: changed pushed to REEF-221

**Pull request closed without merging**